src/EasyAdminBundle/EasyAdminBundle.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
This file is only necessary when you register bundle in AppKernel. This bundle is overriding bundle, not standalone. That means - you don't need to and don't want to register it.

tl;dr - delete this file.